pyramid_oereb/contrib/data_sources/standard/sources/plr.py

```diff
@@ -465,15 +472,12 @@ def collect_legend_entries_by_bbox(self, session, bbox):
         Returns:
             list: The result of the related geometries unique by the public law restriction id
         """
-        distinct_legend_entry_ids = []
-        geometries = self.handle_collection(session, bbox).distinct(
-            self._model_.public_law_restriction_id
-        ).all()
```

Comment:
just for my understanding: why has this become obsolete ?
is there no more possibility of duplicates or are they eliminated later ?